clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -6540,12 +6540,44 @@ void TokenAnnotator::printDebugInfo(const AnnotatedLine &Line) const {
   llvm::errs() << "----\n";
 }
 
+static bool isReturnType(const FormatToken &Tok) {
+  // Look forward to see if there's a function declaration paren
```

Comment:
```suggestion
  // Look forward to see if there's a function declaration paren.
```
Comments end in full stop.